### PR TITLE
Fixes some runtimes + some soft deletions.

### PR DESCRIPTION
**Pull request by PJB3005**
Yes there were 2147 runtimes on **ONE LINE**

### CODE REVIEW

File: code/game/machinery/bots/bots.dm [hunk 1]
```diff
@@ -28,6 +28,11 @@
 			I.bots += src
 	..()
 
+/obj/machinery/bot/Destroy()
+	. = ..()
+	if(botcard)
+		qdel(botcard)
```
Comment (clusterfack) [inline code/game/machinery/bots/bots.dm]: botcard = null after

Comment (PJB3005) [inline code/game/machinery/bots/bots.dm:34]: ah shit right

Comment (PJB3005) [inline code/game/machinery/bots/bots.dm:34]: Alright fixed.

**Comment by 9600bauds** [comment]
:clap:

**Pull request merged**